bin/varnishd/cache/cache_rfc2616.c

```diff
@@ -220,6 +220,16 @@ RFC2616_Ttl(struct busyobj *bo, vtim_real now, vtim_real *t_origin,
 		*grace = rfc2616_time(p);
 	}
 
+	/*
+	 * The "must-revalidate" directive prohibits the cache
+	 * from serving stale data. 
+	 * This sets the grace to zero.
```

Comment:
We should add an RFC comment for coverage too.

Reply:
Just added this and referred to `RFC7234, 5.2.2.1`.

Reply:
We have a special comment format for RFC coverage, see for example:

https://varnish-cache.org/gcov/varnish-cache/bin/varnishd/cache/cache_range.c.html#191

Comments like `// rfc7233,l,455,456` are then referenced:

https://varnish-cache.org/rfc/rfc7233.html#452

Reply:
@Dridi There's no corresponding page for RFC 7234 on https://varnish-cache.org/rfc/. Is that something you can fix?

Reply:
```
curl -s https://tools.ietf.org/rfc/rfc7234.txt | less -N
```

That should help you find the relevant lines covered by your commit. I don't know how @bsdphk set this up but I assume rfc7243 will automagically show up once we start reporting coverage in the master branch.